Trove.js

```diff
@@ -0,0 +1,326 @@
+			}
+
+		}
+	}
+	return "book";
+}
+
+function scrapeWork(doc, url) {
+	var thumbnailURL;
+
+	// Remove all params from url
+	var workURL = url.match(/http.*\/work\/\d+/)[0];
+	var bibtexURL = workURL + '?citationFormat=BibTeX';
+
+	// Need to get version identifier for the BibText url
+	var versionID = doc.body.innerHTML.match(/displayCiteDialog\(\'(.+?)\'/);
+	if (versionID !== null) {
+		bibtexURL += '&selectedversion=' + versionID[1];
+		thumbnailURL = ZU.xpathText(doc, "//a/img[@class='mosaic ui-shdw']/@src");
+	} else {
+		// It's a work -- so thumbnails are different
+		thumbnailURL = ZU.xpathText(doc, "//li[@class='imgfirst']//img/@src");
+	}
+
+	// Get the BibTex and feed it to the translator.
+	ZU.HTTP.doGet(bibtexURL, function(bibtex) {
+		var translator = Zotero.loadTranslator("import");
+		translator.setTranslator("9cb70025-a888-4a29-a210-93ec52da40d4");
+		translator.setString(bibtex);
+		translator.setHandler("itemDone", function (obj, item) {
+			item.itemType = checkType(item.type);
+
+			// This gives a better version-aware url.
+			item.url = ZU.xpathText(doc, "//meta[@property='og:url']/@content");
+			item.abstractNote = ZU.xpathText(doc, "//meta[@property='og:description']/@content");
+			delete item.itemID;
+			delete item.type;
+			if (thumbnailURL !== null) {
+				item.attachments.push({url: thumbnailURL, title: item.title, mimeType:'image/jpeg'});
+			}
+			item.complete();
+		});
+		translator.translate();
+	});
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://trove.nla.gov.au/version/208674371",
+		"items": [
+			{
+				"itemType": "artwork",
+				"title": "THIRROUL - Hotels - Rex Hotel",
+				"creators": [
+					{
+						"firstName": "1910-1981, William A. (William Alan)",
+						"lastName": "Bayley",
+						"creatorType": "author"
+					}
+				],
+				"date": "1960",
+				"abstractNote": "Thirroul Hotels REX HOTEL ca. 1956 1950-1960 Illawarra Region",
+				"libraryCatalog": "Trove",
+				"url": "http://trove.nla.gov.au/version/208674371",
+				"attachments": [
+					{
+						"title": "THIRROUL - Hotels - Rex Hotel",
+						"mimeType": "image/jpeg"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://trove.nla.gov.au/version/11567057",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Experiences of a meteorologist in South Australia",
+				"creators": [
+					{
+						"firstName": "Clement L. (Clement Lindley)",
+						"lastName": "Wragge",
+						"creatorType": "author"
+					}
+				],
+				"date": "1980",
+				"ISBN": "9780908065073",
+				"abstractNote": "In 14 libraries. 24 p. : ill. ; 22 cm. Wragge, Clement L. (Clement Lindley), 1852-1922. South Australia. Climate, 1883-1884. Meteorologists -- South Australia -- Biography. South Australia -- Climate -- History.",
+				"extra": "Reprinted from Good words for 1887/ edited by Donald Macleod, published: London: Isbister and Co",
```

Comment:
which field in Bibtex is this? I'd generally prefer this in a note, but that might require fixing bibtex import rather than here.